unlock-app/src/hooks/useSIWE.tsx

```diff
@@ -118,10 +123,13 @@ export const SIWEProvider = ({ children }: Props) => {
       await queryClient.refetchQueries()
       await refetchSession()
```

Comment:
You need to refactor this signIn into two functions - signInMessage and signIn. The signIn function correctly also re-authentication with our locksmith using the new message which we shouldn't because we wouldn't allow anyone to change that message.

Reply:
good point! We actually need to talk about this more tomorrow during our 1:1 because I realize if locksmith is not running users can't event login which I think we should avoid,